BootloaderCorePkg/Library/FspApiLib/FspSiliconInit.c

```diff
@@ -36,24 +37,26 @@ CallFspSiliconInit (
 
   ASSERT (FspHeader->Signature == FSP_INFO_HEADER_SIGNATURE);
   ASSERT (FspHeader->ImageBase == PcdGet32 (PcdFSPSBase));
+  FspsUpdptr = AllocatePool (FixedPcdGet32 (PcdFSPSUpdSize));
 
+  Status = PcdSet32S (PcdFspsUpdPtr,(UINT32)(UINTN)FspsUpdptr);

```

Comment:
Need add ASSERT_EFI_ERROR (Status); here

Reply:
Let us try to avoid ASSERT here for size reason.   
Since it is a module patchable PCD, it always returns SUCCESS.  No need to do further check.
(VOID) PcdSet32S (...)   should be fine.

Reply:
sure. should no KW issue here if not checking status.